packages/lodestar/src/chain/forkChoice/statefulDag/lmdGhost.ts

```diff
@@ -11,7 +11,7 @@ import {computeSlotsSinceEpochStart,
   getCurrentSlot,
```

Comment:
If you want, you can move the protoarray implementation to a new directory. (Or change the name of the directory)
(This was originally for the "stateful DAG" implementation of the fork choice, now no longer relevant here.)

Reply:
do you want to keep both "stateful DAG" and "protoarray" or just a rename?

Reply:
pro of keeping: might be interesting to be able benchmark one against the other
con of keeping: maintenance of unused fork choice implementation might become a hassle in the future

Lets start with keeping the old implementation. If it turns out we don't need it at all, we can delete it later.